config/locales/en.yml

```diff
@@ -35,14 +35,15 @@ en:
         scheduled: Your mass message has been scheduled.
   voice_response: This phone number can only receive text messages. Please hang up and send a text message.
   messages:
-    empty: "You haven’t sent %{client_first_name} any messages yet. Start by introducing yourself."
-    transferred_from: "%{client_full_name} was transferred to you from %{user_full_name}."
-    transferred_to: "Transferred to %{user_full_name}."
-    phone_number_edited_by_you: "Updated the phone number to %{new_phone_number}"
-    phone_number_edited: "%{user_full_name} updated the phone number to %{new_phone_number}"
+    empty: 'You haven’t sent %{client_first_name} any messages yet. Start by introducing yourself.'
+    transferred_from: '%{client_full_name} was transferred to you from %{user_full_name}.'
+    transferred_to: 'Transferred to %{user_full_name}.'
+    phone_number_edited_by_you: 'Updated the phone number to %{new_phone_number}'
+    phone_number_edited: '%{user_full_name} updated the phone number to %{new_phone_number}'
     admin_user_description: A ClientComm Admin
   message:
-    unclaimed_response: "AUTO REPLY: We don’t recognize this number. Please make sure you include your name and the person you want to reach and we’ll get your message to them."
+    unclaimed_response: 'AUTO REPLY: We don’t recognize this number. Please make sure you include your name and the person you want to reach and we’ll get your message to them.'
+    auto_court_reminder: 'Automated alert: Your next court date is at %{location} on %{date}, %{time} in Rm %{room}. Please text with any questions.'
```

Comment:
We will have to deal with nil or empty room numbers at some point, but it's not a blocker for this chore.